### PR TITLE
Add EIP: Move committee index outside Attestation

### PR DESCRIPTION
**Pull request by dapplion**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-75XX.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+eip: 75XX
```
Comment (g11tech) [inline EIPS/eip-75XX.md:2]: `7549` cc: @abcoathup

File: EIPS/eip-7549.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## Rationale
+
+### Deprecation strategy
+
+The `index` field in `AttestationData` can be deprecated by:
+
+1. Removing the field 
+2. Preserving the field and setting it to be zero
+3. Changing the field type to Optional (from EIP7495 StableContainer)
```
Comment (abcoathup) [inline EIPS/eip-7549.md:49]:
```suggestion
3. Changing the field type to Optional (from EIP-7495 StableContainer)
```
`EIP-N` (not `EIPN` or `EIP N`)

**Review by g11tech — COMMENTED** [review]
looks good apart from a minor change requested for EIP referencing

File: EIPS/eip-7549.md (hunk 2 above)
Comment (dapplion) [reply EIPS/eip-7549.md:49]: When using the dash notation, a linker requested me to link the EIP. How can I do that? Using the full URL returned another linter error

Comment (abcoathup) [inline EIPS/eip-7549.md:49]:
```suggestion
3. Changing the field type to Optional (from [EIP-7495](./eip-7495.md) StableContainer)
```
Sorry, I think the syntax is as above.

**Comment by dapplion** [comment]
CI breaking due to un-releated EIPs. Seem related to the ERC / EIP split

**Review by g11tech — APPROVED** [review]
lgtm

**Pull request merged**